plutus-core/test/Evaluation/DynamicBuiltins/Definition.hs

```diff
@@ -127,51 +128,47 @@ defBuiltinsRuntimeExt
     => BuiltinsRuntime (Either DefaultFun ExtensionFun) term
 defBuiltinsRuntimeExt = toBuiltinsRuntime (defDefaultFunDyn, ()) (defaultCostModel, ())
 
-data ListRep a
+data ListRep (a :: GHC.Type)
```

Comment:
Is this really needed, or you are just trying to be explicit on the kind (for documentation purposes)?

Reply:
We have `PolyKinds` enabled in this file, so that definition might elaborate to `data ListRep (a :: k)`. It also might not, maybe some other definition constrains `a` to be of kind `GHC.Type`. But I didn't want to think about this, so I just specified the right kind.